.github/workflows/python.yml

```
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'python/**'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'python/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9]
```

Note: 3.10 doesn't work due to nose shortcomings.